lib/_http_outgoing.js

```diff
@@ -440,7 +440,8 @@ function storeHeader(self, state, key, value, validate) {
       throw new Error('Header "%s" value must not be undefined', key);
     } else if (checkInvalidHeaderChar(value)) {
       debug('Header "%s" contains invalid characters', key);
-      throw new TypeError('The header content contains invalid characters');
+      throw new TypeError(
+        'The header content contains invalid characters ["' + key + '"]');
```

Comment:
If the error messages are being changed, it would be nice to migrate the files to use internal/errors at the same time.